README.md

```diff
@@ -97,192 +62,117 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+- **Technician endpoint**
+- **Users can add/remove a Technician**
+- **Users can book a Technician**
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+## 💻 Website Mockup <a name="screenshots"></a>
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## Entity Relationship Diagram <a name="er-diagram"></a>
+<img width="850" alt="erd" src="https://user-images.githubusercontent.com/86082070/217819998-c8acf559-359c-4654-99e1-84027a0e6d05.jpeg">
 
-<!-- LIVE DEMO -->
+<!-- Kanban Board -->
+<!-- API Docs -->
+<!-- ER DIAGRAM-->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- [Live Demo]()
+
+## [API Documentation](https://experthub-production.up.railway.app/api-docs/index.html) <a name="api-documentation"></a>
+
+<img width="715" alt="api-docs" src="https://user-images.githubusercontent.com/86082070/217821271-c90e1a90-e81d-4ed2-b4ce-08e9d960a7e4.png">
+
+<img width="703" alt="api-docs" src="https://user-images.githubusercontent.com/86082070/217821444-053f5958-6983-4105-a562-2860e98cebf7.png">
+
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
-
 ### Prerequisites
 
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
+Make sure you have `Git` and `Ruby` installed
 ```
- -->
-
-### Setup
-
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+ gem install rails
 ```
---->
 
 ### Install
 
-Install this project with:
+Install the project using this command:
 
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
 ```
---->
+  bundle install
+```
 
 ### Usage
 
-To run the project, execute the following command:
-
-<!--
-Example command:
+Run project using this command:
 
-```sh
-  rails server
 ```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
+rails s
 ```
---->
-
-### Deployment
-
-You can deploy this project using:
-
-<!--
-Example:
-
-```sh
 
+Run tests using this command:
+```
+rspec spec
```

Comment:
- [ ] When running tests according to your readme instructions, this happens:

![image](https://user-images.githubusercontent.com/57421823/217851192-61180a67-926d-4ef6-bd40-1ebcff903af4.png)

This is because all your tests are unit tests and use swagger.  There are 2 possible solutions for this:

a) Change your test command to `rake rswag`
b) Add some unit tests, and be specific about how to run them in the terminal, along with the swagger ones.

That way, your users will never get a failed test. 💪